Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/UserManager.java

```diff
@@ -297,29 +293,31 @@ public int activateUser(String sUserId, String sAdminId) throws QuadrigaStorageE
      * 
      * @param request
      *            The {@link AccountRequest} encapsulating the user's data.
-     * @return true if request was successfully added; otherwise false
+     * @return true if request was successfully added; false if username exists or the account needs to be approved by the admin.
      * @author jdamerow
      * @throws QuadrigaStorageException
-     * @throws UsernameExistsException
      * @throws QuadrigaNotificationException 
      */
     @Override
     @Transactional
-    public boolean addNewUser(AccountRequest request) throws QuadrigaStorageException, UsernameExistsException {
+    public boolean addNewUser(AccountRequest request) throws QuadrigaStorageException{
         QuadrigaUserDTO userDTO = usermanagerDAO.getUserDTO(request.getUsername());
 
         // Check if username is already in use
-        if (userDTO != null)
-            throw new UsernameExistsException("Username already in use.");
-
+        if (userDTO != null){
+            logger.warn("Username already in use or user account needs to be approved by the admin.");
+            return false;
+        }
```

Comment:
this method needs to through an exception here. If addNewUserAccountRequest wasn't successful, it'll return false. Doesn't mean that the username already exists. In this scenario, throwing an exception is the only way to indicate what went wrong.